requirements.txt

```diff
@@ -1,12 +1,37 @@
+appdirs==1.4.0
+autoflake==0.6
+autopep8==1.2
```

Comment:
Yeah, we don't need these requirements, although, we could just add `cib` and add a script line `cib install -c .coafile`